source/lib/course-search/parse-term.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+export function parseTerm(term: string) {
+	const semester = parseInt(term.slice(-1, term.length))
```

Comment:
Use `term.slice(-1)` please

```
$ node
> '20161'.slice(-1)
'1'
```

Reply:
Additionally, please remove the `parseInt` from this, and just use strings in the switch statement below.